azure-quantum/azure/quantum/aio/target/quantinuum.py

```diff
@@ -0,0 +1,42 @@
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+from typing import Any, Dict
+
+from azure.quantum.aio.target.target import Target
+from azure.quantum.aio.job.job import Job
+from azure.quantum.aio.target import Honeywell
+
+
+class Quantinuum(Honeywell):
+    """Quantinuum target."""
+
+    async def submit(
+        self,
+        circuit: str,
+        name: str = "quantinuum-job",
+        num_shots: int = None,
+        input_params: Dict[str, Any] = None,
+        **kwargs
+    ) -> Job:
+        """Submit a Quantinuum program (QASM format)
```

Comment:
QASM on its own is ambiguous, as it could refer to OpenQASM 2.0, OpenQASM 3.0, cQASM, eQASM, or even the older qasm2circ format.

```suggestion
        """Submit a Quantinuum program (OpenQASM 2.0 format)
```